index.d.ts

```diff
@@ -957,6 +957,7 @@ declare namespace Moleculer {
 		tracking?: boolean;
 		paramsCloning?: boolean;
 		caller?: string;
+		settled?: boolean;
```

Comment:
This new property is available only in `mcall` method and not for other places where `CallingOptions` is used. Please define a new interfaces for `mcall` with this property.

Reply:
I've just pushed a new version

Reply:
@dprslt please, remove `settled` from `CallingOptions`